app-tasks/dags/export/do_export.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import print_function
+import datetime
+import logging
+import os
+import time
+import json
+
+import subprocess
+
+from airflow.models import DAG
+from airflow.operators import BashOperator, PythonOperator
+from airflow.exceptions import AirflowException
+
+from rf.utils.io import get_session
+from rf.utils.exception_reporting import wrap_rollbar
+
+
+rf_logger = logging.getLogger('rf')
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+rf_logger.addHandler(ch)
+
+HOST = os.getenv('RF_HOST')
+API_PATH = '/api/exports/'
+
+logger = logging.getLogger(__name__)
+
+default_args = {
+    'owner': 'raster-foundry',
+    'start_date': datetime.datetime(2017, 1, 1)
+}
+
+dag = DAG(
+    dag_id='do_export',
+    default_args=default_args,
+    schedule_interval=None,
+    concurrency=os.getenv('AIRFLOW_DAG_CONCURRENCY', 4)
+)
+
+
+
+# UTILS
+@wrap_rollbar
+def update_status(export, status):
+    url = '{0}{1}{2}/'.format(HOST, API_PATH, export)
```

Comment:
Sorry that the "Notes" item didn't make it clearer, but I also would prefer to handle the status updating in the JAR, because the "running state" of the task, as well as any possible additional handling, would be much clearer within the `rf-batch` instance than from any CL output sent back to airflow.

I didn't implement this workflow as such because I wasn't sure where the proper balance was between airflow task and executable task so I deferred to the current pattern. But if it falls within the scope of this issue I can push a revision to update status as part of the executable.